src/options.cpp

```diff
@@ -1030,6 +1030,13 @@ void options_manager::init()
 
     mOptionsSort["interface"]++;
 
+    add( "DIAG_MOVE_WITH_MODIFIERS", "interface", translate_marker( "Diagonal movement with cursor keys and modifiers" ),
+        translate_marker( "If true, allows diagonal movement with cursor keys using CTRL and SHIFT modifiers (<color_red>currently only in SDL builds</color>).  Diagonal movement action keys are taken from keybindings, so you need these to be configured." ),
```

Comment:
No other build specific option mentions that is only available in certain build types. (Note that `COPT_CURSES_HIDE` already hides it on curses builds.) For the sake of consistency, this should not be mentioned at all, or be mentioned by all options, preferably automated by checking `cOpt::hide`.

Reply:
I've removed SDL build mention from option description.